pipelines/controller/src/main/java/com/google/fhir/analytics/PipelineManager.java

```diff
@@ -211,15 +220,16 @@ private void initDwhStatus() throws ProfileException {
               prefix,
               DwhFiles.TIMESTAMP_PREFIX);
         }
-        if (lastDwh.isEmpty() || lastDwh.compareTo(path.getFilename()) < 0) {
+        if (lastDwh != null && (lastDwh.isEmpty() || lastDwh.compareTo(path.getFilename()) < 0)) {
```

Comment:
I think `lastDwh` should always be non-null; so instead of this, I think we can add a condition for `path.getFilename()` not being null before assigning it to `lastDwh`.